RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -114,8 +114,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/radixdlt/sargon",
       "state" : {
-        "revision" : "30cfe39a9e108ac2e9c7a4f01fb917ba131869a1",
-        "version" : "0.7.19"
+        "revision" : "60006952864e2b4fb1e511a83844694819132d04",
+        "version" : "1.0.1"
```

Comment:
🎉 I chose version `1.0.x` of Sargon to mark the first wallet release to use Sargon Profile 🥳